devel/qbs/Portfile

```diff
@@ -32,10 +34,10 @@ compiler.cxx_standard 2017
 # https://github.com/macports/macports-base/pull/179
 compiler.blacklist-append {clang < 1001}
 
-configure.post_args QBS_INSTALL_PREFIX=${prefix} \
-                    CONFIG+=qbs_disable_rpath \
-                    CONFIG+=qbs_enable_project_file_updates \
-                    qbs.pro
+configure.args-append -DQBS_ENABLE_RPATH=NO
+
+cmake.build_type    Release
```

Comment:
can this be left to the default "MacPorts" value so that the correct/default flags are used?